frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -187,11 +187,11 @@ const ClusterVersionDetailsTable: React.SFC<ClusterVersionDetailsTableProps> = (
         </div>
       </div>
       <div className="co-m-pane__body-group">
-        <dl className="co-m-pane__details">
+        <dl className="co-m-pane__details" >
           <dt>Cluster ID</dt>
-          <dd className="co-break-all co-select-to-copy">{getClusterID(cv)}</dd>
+          <dd className="co-break-all co-select-to-copy" data-test-id="cv-details-table">{getClusterID(cv)}</dd>
           <dt>Desired Release Image</dt>
-          <dd className="co-break-all co-select-to-copy">
+          <dd className="co-break-all co-select-to-copy" data-test-id="cv-details-table">
```

Comment:
I would give each element a different `data-test-id`. For instance,

```suggestion
          <dd className="co-break-all co-select-to-copy" data-test-id="cv-desired-image">
```

etc